<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Console</class>
 <widget class="QDialog" name="Console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件学院</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>90</y>
     <width>831</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAdd1_1">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>121</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSubtract1_1">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>121</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtract</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leGoal1_1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReset1_1">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>120</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSubtract1_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>171</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtract</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReset1_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>170</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAdd1_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>171</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leGoal1_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSubtract1_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>221</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtract</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReset1_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>220</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAdd1_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>221</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leGoal1_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSubtract1_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>271</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtract</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReset1_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>270</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAdd1_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>271</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leGoal1_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSubtract1_5">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>321</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtract</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReset1_5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>320</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAdd1_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>321</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leGoal1_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件学院</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSubtract1_6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>371</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtract</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReset1_6">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>370</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAdd1_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>371</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leGoal1_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>370</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件学院</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>第一环节</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Next">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Last">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Clear">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>70</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 繁黑體 Std B</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Show">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 繁黑體 Std B</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Result">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ShowAgain</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Hide">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hide</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_SetAll">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 繁黑體 Std B</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SetAll</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
